spec/models/pairing_spec.rb

```diff
@@ -22,5 +22,29 @@
         expect(pairing_3.cancelled?).to eq(true)
       end
     end
+    describe 'create pairing' do
+      it 'can create a pairing' do
+        japanese_learning_english = create(:user)
+        english_learning_japanese = create(:user)
+        availability = japanese_learning_english.availabilities.create(start_date_time: DateTime.new(2021, 1, 1, 14, 30), end_date_time: DateTime.new(2021, 1, 1, 15, 30), status: 0)
+        open_slot = english_learning_japanese.availabilities.create(start_date_time: DateTime.new(2021, 1, 1, 13, 30), end_date_time: DateTime.new(2021, 1, 1, 15, 30), status: 0)
+        
+        Pairing.create_pairing(availability, [open_slot])
+
+        expect(Pairing.all.size).to eq(1)
+      end
+    end
+    describe 'determine_time_of_pairing' do
```

Comment:
can we add an extra line here and wrap in a describe class methods group? Main now has a section for instance pairings

Reply:
Done

Reply:
Did you add the class method section? @lkriffell

Reply:
Ohh I see what you mean now, yeah I can do that. I had a conflict and added them to the instance methods describe